src/utils/index.ts

```diff
@@ -59,6 +59,6 @@ export const findDOMNode = <E extends Element = Element>(
 type ReactText = string | number;
 type _ReactChild = React.ReactElement<unknown> | ReactText;
 
-interface ChildrenArray extends Array<Children> {}
-type ReactFragment = ChildrenArray;
+export interface ChildrenArray extends Array<Children> {}
+export type ReactFragment = ChildrenArray;
```

Comment:
why these exports?

Reply:
TS is otherwise complaining with:
<img width="622" alt="Screenshot 2019-09-17 at 19 58 27" src="https://user-images.githubusercontent.com/2643520/65066898-97abb080-d985-11e9-8436-398be346f8e8.png">
I didn't really investigate this thoroughly 🤷‍♂

Reply:
actually, it goes away if I use the `private` modifier for this method

Reply:
but I can't just use the same "fix" for `render()` here:

<img width="676" alt="Screenshot 2019-09-17 at 20 47 22" src="https://user-images.githubusercontent.com/2643520/65070170-6b476280-d98c-11e9-9648-553f9da32fc6.png">

So I'd leave it as is now